Support `fields:` option to skip selector query (#20)
github/session-resume: README.md

The `persistResumableFields(id: string, options)` function supports optional configurations:

* `storage:` - [`Storage`][] instance (defaults to [`window.sessionStorage`][])
* `storageFilter:` - `(field: HTMLInputElement | HTMLTextAreaElement) => boolean` predicate to determine whether or not to store a field (defaults to `(field) => field.value !== field.defaultValue`)
* `keyPrefix:` - `string` prepended onto the storage key (defaults to `"session-resume"`)
* `scope:` - `ParentNode` used to query field elements (defaults to `document`)
* `selector:` - `string` used to query field elements (defaults to `".js-session-resumable"`)
* `fields:` - `NodeList | Node[]` provide field elements as an alternative to querying (defaults to `options.scope.querySelectorAll(options.selector)`)

> **Note:** When combined, values passed as `fields:` will override options passed as `scope:` and `selector:`.

Reviewer: I think this is a bit unclear... how about:
```suggestion
**Note:** When `fields` is specified, `scope` and `selectors` will be ignored.
```
